src/analyze/BuildSystem.re

```diff
@@ -223,8 +223,18 @@ let getStdlib = (base, buildSystem) => {
     /+ "lib"
     /+ "ocaml"]
   | Dune(Esy) =>
-    let%try_wrap esy_ocamllib = getLine("esy -q sh -- -c 'echo $OCAMLLIB'", ~pwd=base);
-    [esy_ocamllib]
+    let env = Unix.environment()->Array.to_list;
+    try {
```

Comment:
I'd rather avoid such large `try` blocks. You're just dealing with the `Not_found` from `List.find`, right? Could you use `Belt.List.getBy` instead which returns an optional instead of throwing?